sass/_theme_layout.sass

```diff
@@ -370,12 +371,15 @@ footer
     display: block
   .wy-nav-side
     @if $nav-desktop-position == left
-      left: -$nav-desktop-width
+      transform: $sidebar-transform-hide
+      transition: $sidebar-transition
     @else
```

Comment:
The `$nav-desktop-position` is already broken without sliding, no need to worry about this line I think